GVFS/GVFS.FunctionalTests/Should/FileSystemShouldExtensions.cs

```diff
@@ -276,10 +308,16 @@ public DirectoryInfo WithAttribute(FileAttributes attribute)
 
                 while (expectedMoved && actualMoved)
                 {
+                    if (withinPrefixes != null)
+                    {
+                        AdvanceUntilMatchingPrefix(expectedEnumerator, expectedPath, withinPrefixes);
+                        AdvanceUntilMatchingPrefix(actualEnumerator, actualPath, withinPrefixes);

```

Comment:
This "actual" should be dropped, since we really do want to only have things inside the cone, anyway.

Reply:
If things exist outside of the cone, I want to know about it. I'll leave this here until I have passing tests, then drop it and react to any tests that may fail because we get new folders.